docs/current_docs/integrations/snippets/argo-workflow.yaml

```diff
@@ -18,13 +18,15 @@ spec:
           - name: project-source
             path: /work
             git:
+              # replace with your repository URL
               repo: YOUR_REPOSITORY_URL_HERE
               revision: "main"
           - name: dagger-cli
             path: /usr/local/bin
             mode: 0755
             http:
-              url: https://github.com/dagger/dagger/releases/download/v0.10.2/dagger_v0.10.2_linux_arm64.tar.gz
```

Comment:
:eyes: I don't think we should point people to github to download the CLI - we should point people to `dl.dagger.io`, right?

Reply:
+1 we should follow the same pattern as here: https://github.com/dagger/dagger/blob/main/docs/current_docs/integrations/snippets/circle.yml#L16-L17

I believe @kpenfound has an example about this maybe?

Reply:
@kpenfound can you suggest how to change this?

Reply:
Looks like input artifacts only support git, HTTP, GCS, and S3, but not scripts. So the alternative would be to run a script as another step, which isn't great considering this really should be an input artifact IMO.
I'm good with the current state of this PR